test/jasmine/tests/histogram2d_test.js

```diff
@@ -157,6 +170,60 @@ describe('Test histogram2d', function() {
                 [0, 0, 0, 1]
             ]);
         });
+
+        ['histogram2d', 'histogram2dcontour'].forEach(function(traceType) {
+            it('@flaky should sort z data based on axis categoryorder for ' + traceType, function() {
```

Comment:
@antoinerg do you have idea why this needs `@flaky` ? I mean `_module.calc` routines _can't_ be flaky right?

Reply:
@etpinard I am puzzled by this. Locally the tests run fine but on the CI it would keep failing. Commit 22cfa82 (prior to adding flaky) just wouldn't pass `test-jasmine` on CircleCI and it failed with the same error 4-5 times in a row...

Reply:
On CircleCI, it seems to consistently fail with:
![Screenshot_2019-05-06_12-27-23](https://user-images.githubusercontent.com/301546/57239599-5334ff00-6ffa-11e9-91b9-94aee0325f10.png)
:man_shrugging:

Reply:
I could remove the "flaky" tag with commit 4d4b0b748d71fb6ef272369de06f4d00c6e45e43 ! :tada: